contracts/finality/src/contract.rs

```diff
@@ -180,6 +180,8 @@ pub(crate) mod tests {
             admin: init_admin.to_string(), // Admin provided
             bsn_id: "op-stack-l2-11155420".to_string(),
             min_pub_rand: 100,
+            max_msgs_per_interval: MAX_MSGS_PER_INTERVAL,
```

Comment:
Should we test setting invalid values for those?

Reply:
We should, and it's under `msg.rs` now